lib/db.js

```diff
@@ -3,13 +3,30 @@ const levelup = require('level');
 const db = Promise.promisifyAll(levelup('./data'));
 
 exports.scores = {
-	get: function(id) {
-		return db.getAsync(id)
+	get: function(id, rev) {
+		return db.getAsync(id + ':' + rev)
 			.then(JSON.parse)
 	},
+	revisions: function(id) {
+		return db.getAsync(id + ':revisions')
+			.then(null, function (err) {
```

Comment:
`.then(function(r) { return parseInt(r, 10); }, ...`

Better yet, tell LevelUP to treat everything as JSON. https://github.com/Level/levelup#json-data